webscraper/dataScraperAPI.js

```diff
@@ -10,65 +10,221 @@ function scrape() {
   let currSwellInfo = {};
   let histSwellInfo = {};
 
-  const response = axios(url).then(response => {
-    const html = response.data;
-    const $ = cheerio.load(html);
-    // Scraping info for currWindInfo
-    currWindInfo["windDirection"] = $("#data > table:nth-child(2) > tbody:nth-child(2) > tr:nth-child(2) > td:nth-child(3)").text().trim();
-    currWindInfo["windSpeed"] = $("#data > table:nth-child(2) > tbody:nth-child(2) > tr:nth-child(3) > td:nth-child(3)").text().trim();
+  const response = axios(url)
```

Comment:
Refine the data structure to avoid the need to reference values by index number. Consider using the columnName as a key in every row, so that would make the data an array of objects with keys representing the columnName or label.